File: app/src/main/java/com/ishow/noahark/AppApplication.java
package com.ishow.noahark;

import android.app.Application;

import com.ishow.common.utils.http.rest.Http;
import com.ishow.common.utils.image.loader.ImageLoader;
import com.ishow.common.utils.router.AppRouter;
import com.ishow.noahark.utils.http.AppHttpCallBack;
import com.ishow.noahark.utils.router.AppRouterConfigure;

/**
 * Application
 */
public class AppApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        Http.init(this);
        ImageLoader.init(this);

        AppRouter.setConfigure(new AppRouterConfigure());
    }
}

File: app/src/main/java/com/ishow/noah/AppApplication.kt
package com.ishow.noah

import android.app.Application
import com.facebook.drawee.backends.pipeline.Fresco
import com.ishow.common.utils.router.AppRouter
import com.ishow.noah.utils.router.AppRouterConfigure

/**
 * Application
 */
class AppApplication : Application() {
    override fun onCreate() {
        super.onCreate()

        Fresco.initialize(this)
        AppRouter.setConfigure(AppRouterConfigure())
    }
}
